caliban_toolbox/__init__.py

```diff
@@ -23,4 +23,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Data Engineering Toolbox for DeepCell"""
+"""Data Engineering Toolbox for Caliban"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from caliban_toolbox import pre_annotation
+# from caliban_toolbox import tracking
+from caliban_toolbox import utils
+from caliban_toolbox import aws_functions
+from caliban_toolbox import figure_eight_functions
+from caliban_toolbox import log_file
+from caliban_toolbox import relabel
+from caliban_toolbox import reshape_data
+
+from caliban_toolbox.pre_annotation import *
+# from caliban_toolbox.tracking import *
+from caliban_toolbox.utils import *
```

Comment:
Prefer to upload by name, or else it's possible to import 3rd party packages from that module.

You can also use `__all__` in the file to explicitly define what is able to be imported from a file.

Reply:
As you no doubt noticed, I didn't include any imports for anything. I'm not sure I quite understand the point of doing this? Is it good software practice to explicitly define what can be imported to avoid having things you don't want accessible? Or is there some other reason that it's good to include these?

Reply:
Yeah its just improving/cleaning the namespace of the API. Its weird to have `caliban.pre_annotation.skimage` in the namespace.

Reply:
will adjust in future PR when the utils and data loader functions are more set